### ISSUE TITLE
Item Router nonfunctional

### ISSUE THREAD

**Issue by ErnestoPeroPezzi** [BluSunrize/ImmersiveEngineering#1835]
the item router does not work at all, I tested it thoroughly, and the conclusion is that it spits out items wherever it pleases regardless of the set filters or not. 

Also, the table looks like this and has looked like that for a while now and I never realized thats not how its supposed to look.
[https://gyazo.com/bb24e68b63b624237faa544f23346db4](url)

If by off chance im just plain stupid, what is the proper way to use the item router? The things you put in correspond as a whitelist for those items ONLY to pass through its corresponding colour correct?

**Comment by ErnestoPeroPezzi** [comment]
https://gyazo.com/bb24e68b63b624237faa544f23346db4

**Comment by sphynx1492** [comment]
We are having the same issue as well.

**Comment by bookerthegeek** [comment]
It took me a moment to realize that they all point down. 

In not having this issue. 

What version are you using?

**Comment by BluSunrize** [comment]
I believe the issue here is that the router had it's rotation, as well as GUI, messed up.
The white side is supposed to be Up, and the table on the inside is lettered:
D N W
T S E
With that in mind, it may have been outputting correctly. For example, if you put Lapis Lazuli in the green grid, it should come out the west side. I will be fixing the rotations in the next update, they were introduced with the addition of the Turntable and it's subsequent changes to the blockstate file, and the GUI was a localization error that will be fixed as well.

The routing, however, works in my dev environment, so please provide further info on any troubles with it you experience.

**Comment by sphynx1492** [comment]
We are using the latest 0.10-54.  Just to give a little more information.  I have fixed mine for the time being, I believe.  Everything seems to be working now.  I was trying to use the filter to get an item to go out one direction, and then leaving the rest empty and just let it "auto detect" the output to the other conveyor belt.  When I used it like this, it would allow that item to go out both directions, so I went back and changed the filter to only allow the rest of the materials through.  It took a little more configuring on the filters than I had thought, but I believe all is working now.
